assets/common.sh

```diff
@@ -13,8 +13,18 @@ start_docker() {
       mount -n -t cgroup -o $d cgroup /sys/fs/cgroup/$d
   done
 
+  for i in $(seq 0 100) ; do
+    mknod -m 660 /dev/loop${i} b 7 ${i}
+  done
+
   mkdir -p /var/lib/docker
-  mount -t tmpfs -o size=100G none /var/lib/docker
+  image=$(mktemp $PWD/docker.img.XXXXXXXX)
+  dd if=/dev/zero of=${image} bs=1 count=0 seek=100G
+  mkfs.ext4 -F ${image}
+  losetup -f ${image}
```

Comment:
Unfortunately, no; busybox `losetup` doesn't support `--show`.  I actually tried that one first because I was following the [code in baggageclaim][btrfs.go] you suggested. :|  This (and the random filename) was all to workaround that lack.

[btrfs.go]: https://github.com/concourse/baggageclaim/blob/7fdf04194a2e8fa28e26c03a03756f7f3c389655/fs/btrfs.go#L33-L51